frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/Summary2OverrideEntry.tsx

```diff
@@ -85,6 +105,24 @@ export const Summary2OverrideEntry = ({
           onChangeOverride('emptyFieldText', event.target.value)
         }
       ></StudioTextfield>
+      {override.componentId && checkboxOrMultipleselect && (
+        <Summary2OverrideDisplaytype override={override} onChange={onChange} />
+      )}
+      {isComponentInGroup && (
+        <Checkbox
+          size='sm'
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+            onChangeOverride(
+              event.target.value as keyof Summary2OverrideConfig,
+              event.target.checked,
+            )
+          }
+          checked={override.isCompact ?? false}
+          value={'isCompact'}
+        >
+          {t('ux_editor.component_properties.overrides_is_compact')}
+        </Checkbox>
+      )}
```

Comment:
What do you think about refactoring this out into a separate component at the bottom of the file (no need to export it)? 😄 Then we have one component that does one thing only, and the code is split into smaller chunks 😄 
Something like this maybe? 

You can call it like this: 
```tsx
{isComponentInGroup && (
   <ComponentInGroupCheckbox onChangeOverride={onChangeOverride} override={override} />
)}
```

You can have the component at the bottom of the file like this: 
```tsx
type ComponentInGroupCheckboxProps = {
   onChangeOverride = (label: keyof Summary2OverrideConfig, value: string | boolean) => void;
   override: Summary2OverrideConfig;
}

const ComponentInGroupCheckbox = ({
   onChangeOverride, 
   override
}): ReactElement => {
   const { t } = useTranslation();
   
   const handleChange = (event: ChangeEvent<HTMLInputElement) => {
      onChangeOverride(
         event.target.value as keyof Summary2OverrideConfig,
         event.target.checked,
      )
   };

   return (
      <Checkbox
          size='sm'
          onChange={handleChange}
          checked={override.isCompact ?? false}
          value='isCompact'
        >
          {t('ux_editor.component_properties.overrides_is_compact')}
        </Checkbox>
   )
};
```